### ISSUE TITLE
Enter vs shift enter -  <br> vs <pr>

### ISSUE THREAD

**Issue by Silvernerd1** [espanso/espanso#1712]
Hi everyone,

when I code "aaa\nbbb" Espanso simulates pressing the [Enter] key. This means "create a new paragraph" in many systems as MS Word or HTML (<p> instead of <br>). The same happens when I code
- trigger: ":ab"
  replace: |
    aaa
    bbb

By chance I found out, that in some cases, a [Shift] + [Enter] is simulated, especially, when I code long lines:
- trigger: ":long"
  replace: |
    This is a very very long line 01. This is a very very long line 01. This is a very very long line 01. 
    This is a very very long line 02. This is a very very long line 02. This is a very very long line 02. 
    This is a very very long line 03. This is a very very long line 03. This is a very very long line 03. 

But I couldn't figure out exactly, in which cases [Enter] / <pr> is produced and in which cases [Shift] + [Enter] / <br>

I checked the Espanco documentation and the YAML documentation either. Nothing found about this issue.

Is this behaviour random or are there any rules?

I use Espanso 2.1.8 on Windows 11 professional.

Bfn
Michael

**Comment by smeech** [comment]
I can't duplicate the issue here (Mint Xfce, LibreOffice Writer) but it may relate to the `clipboard_threshold` setting and the changeover of `backend`s. See [Options reference](https://espanso.org/docs/configuration/options/#options-reference).

**Comment by Silvernerd1** [comment]
Thank you for your posting, smeech.

Meanwhile, i figured out, that this syntax works fine for me:
`- trigger: ".br"
  **force_clipboard: true**   
  replace: |
   Best regards
   Michael
`

**Comment by smeech** [comment]
Yes, that does much the same - in fact I have so many of these in scripts and app-specific configurations I'm beginning to wonder what the advantages of using `inject`, instead, are.

**Comment by smeech** [comment]
Please note, `force_clipboard:` is now deprecated in favour of the `force_mode: clipboard`/`keys` options, which are more flexible.
See #1939.
